src/Logging/Logging.AzureAppServices/src/Internal/BatchingLoggerProvider.cs

```diff
@@ -107,7 +118,10 @@ internal void AddMessage(DateTimeOffset timestamp, string message)
             {
                 try
                 {
-                    _messageQueue.Add(new LogMessage { Message = message, Timestamp = timestamp }, _cancellationTokenSource.Token);
+                    if (!_messageQueue.TryAdd(new LogMessage { Message = message, Timestamp = timestamp }, millisecondsTimeout: 0, cancellationToken: _cancellationTokenSource.Token))
```

Comment:
Is the `millisecondsTimeout: 0` needed for this change?

Reply:
https://docs.microsoft.com/en-us/dotnet/api/system.collections.concurrent.blockingcollection-1.tryadd?view=netframework-4.7.2#System_Collections_Concurrent_BlockingCollection_1_TryAdd__0_System_Int32_System_Threading_CancellationToken_

It's not an optional parameter in this overload.